cinch/roles/jenkins_master/templates/enable-kerberos-sso.groovy

```diff
@@ -0,0 +1,21 @@
+def plugin = com.sonymobile.jenkins.plugins.kerberossso.PluginImpl.getInstance()
```

Comment:
It probably should. I don't understand anything about what it's doing, though, so I left this one unedited.